src/api/docker-files/Dockerfile

```diff
@@ -3,11 +3,19 @@
 # contained rails app generating files in the git checkout with
 # some strange user...
 
-FROM openbuildservice/frontend-base
+FROM registry.opensuse.org/obs/server/unstable/container/leap151/containers/openbuildservice/frontend-base
```

Comment:
The container on docker hub is "just" a gem cache and rebuilds every time we commit. The container on OBS doesn't.

Reply:
Ah you do already, sorry